JuiceMaker/JuiceMaker/Controller/JuiceMakeViewController.swift

```diff
@@ -7,14 +7,103 @@
 import UIKit
 
 final class JuiceMakeViewController: UIViewController {
-
+    
+    let fruitStore = FruitStore.shared
+    
+    var currentFruitBasket: [Fruit: Int] {
+        return FruitStore.shared.shareFruitBasket()
+    }
+    
+    let juiceMaker = JuiceMaker()
+    
+    @IBOutlet weak var strawberryLabel: UILabel!
+    @IBOutlet weak var bananaLabel: UILabel!
+    @IBOutlet weak var pineappleLabel: UILabel!
+    @IBOutlet weak var kiwiLabel: UILabel!
+    @IBOutlet weak var mangoLabel: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
-        let juiceMaker = JuiceMaker()
-        juiceMaker.make(.mangoKiwiJuice)
+        setUpLabel()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        setUpLabel()
+    }
+    
+    func setUpLabel() {
+        strawberryLabel.text = currentFruitBasket[.strawberry]?.description
+        bananaLabel.text = currentFruitBasket[.banana]?.description
+        pineappleLabel.text = currentFruitBasket[.pineapple]?.description
+        kiwiLabel.text = currentFruitBasket[.kiwi]?.description
+        mangoLabel.text = currentFruitBasket[.mango]?.description
+        [strawberryLabel, bananaLabel, pineappleLabel, kiwiLabel, mangoLabel].forEach {
+            $0.sizeToFit()
+        }
+    }
+    
+    func buttonTarget(_ sender: UIButton) -> FruitJuice? {
+        guard let titleLabel = sender.titleLabel?.text else {
+            return nil
+        }
+        
+        switch titleLabel {
+        case "딸기쥬스 주문":
+            return FruitJuice.strawberryJuice
+        case "바나나쥬스 주문":
+            return FruitJuice.bananaJuice
+        case "파인애플쥬스 주문":
+            return FruitJuice.pineappleJuice
+        case "키위쥬스 주문":
+            return FruitJuice.kiwiJuice
+        case "망고쥬스 주문":
+            return FruitJuice.mangoJuice
+        case "딸바쥬스 주문":
+            return FruitJuice.strawberryBananaJuice
+        case "망키쥬스 주문":
+            return FruitJuice.mangoKiwiJuice
+        default:
+            return nil
+        }
     }
+    
+    func showSuccessAlert(with targetJuice: FruitJuice) {
+        let successAlert = UIAlertController(title: "주스 제조성공", message: "\(targetJuice.rawValue) 나왔습니다! 맛있게 드세요!", preferredStyle: .alert)
+        let alertAction = UIAlertAction(title: "OK", style: .default)
+        successAlert.addAction(alertAction)
+        self.present(successAlert, animated: true)
+    }
```

Comment:
UIAlertController 를 생성해주는 메서드이군요 👍
```suggestion
    func showSuccessAlert(with targetJuice: FruitJuice) {
        let successAlert = UIAlertController(
            title: "주스 제조성공",
            message: "\(targetJuice.rawValue) 나왔습니다! 맛있게 드세요!", 
            preferredStyle: .alert
        )
        let alertAction = UIAlertAction(title: "OK", style: .default)
        successAlert.addAction(alertAction)
        self.present(successAlert, animated: true)
    }
```

파라미터가 3개이상일 때 이런 식으로 개행을 해주기도 합니다. 스타일이니 참고만 해주세요 !☺️

Reply:
개행했습니다! 추가적으로 destructive옵션을 cancle로 바꿔주어 데이터삭제를 암시하는 것이 아닌 단순취소만을 생각할 수 있도록 변경했습니다!